packages/notebook/src/searchprovider.ts

```diff
@@ -721,7 +721,7 @@ export class NotebookSearchProvider extends SearchProvider<NotebookPanel> {
       // change, and if selection is getting extended, we do not want to clear
       // highlights just to re-apply them shortly after, which has side effects
       // impacting the functionality and performance.
-      this._delayedActiveCellChangeHandler = setTimeout(() => {
+      this._delayedActiveCellChangeHandler = window.setTimeout(() => {
```

Comment:
This is unrelated to this PR but it was causing the `watch` compiler to fail so I updated it.